docs/how-to/report-security-vuln.rst

```diff
@@ -0,0 +1,57 @@
+===============================
+Report security vulnerabilities
+===============================
+
+
+How to report
```

Comment:
Also, no need to for this H2.

Moreover, we don't recommend following a suheading directly after a heading without content. (Headings require content and should not be followed directly by a subheading - See [style guide](https://docs.ubuntu.com/styleguide/en#headings).